src/main/webapp/style.css

```diff
@@ -1,7 +1,41 @@
+body, html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
+.card {
+  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
+  position: relative;
+  transition: box-shadow .25s;
+  border-radius: 2px;
+}
+
+.card .card-title {
+  font-size: 18px;
+  font-weight: 300;
+  cursor: pointer;
+  line-height: 32px;
+  display: flex;
+}
+
+.card .card-content {
+  padding: 24px;
+  border-radius: 0 0 2px 2px;
+}
+
+.card .card-content p {
+  display: flex;
+  line-height: 32px;
+  margin: 0;
+}
+
 h3 {
   text-align: center;
 }
 
 .srp-container {
   display: flex;
 }
+
+.srp-child {
+  flex: 0 0 50%;
+}
```

Comment:
style nit: I think it would look better if it was more 2/3 search results, 1/3 card filling up the page rather than 50/50 right now

style nit: Add margin to .card so that the cards don't cling to the sides of the page.